.github/workflows/benchmarks.yml

```diff
@@ -46,18 +46,18 @@ jobs:
 
       - name: "Install lowest dependencies"
         if: ${{ matrix.dependencies == 'lowest' }}
-        run: "composer update --prefer-lowest --no-interaction --no-progress --no-suggest"
+        run: "composer update --prefer-lowest --root-reqs --no-interaction --no-progress"
 
       - name: "Install highest dependencies"
         if: ${{ matrix.dependencies == 'highest' }}
-        run: "composer update --no-interaction --no-progress --no-suggest"
+        run: "composer update --no-interaction --root-reqs --no-progress"
```

Comment:
We should really not use `--root-reqs` when testing upgrades/downgrades: what users get is a full "lowest" or "newest", not just dependencies we declared.

For our own stability needs, there is the `locked` pipeline instead.